container_pipeline/lib/openshift.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import time
+
+from container_pipeline.lib import settings
+
+from .lib.command import run
+
+
+class OpenshiftError(Exception):
+    pass
+
+
+class Openshift(object):
```

Comment:
Python API will take more development time than using system commands. Most of the time, I have seen that libraries don't have 1-1 mapping with the command line. That said, it's good to have, and we can do it sometime. Since, we have abstracted out Openshift in a class, it will be easy to refactor without breaking other code.